docker/service/src/test/java/uk/nhs/adaptors/scr/models/GpSummaryTest.java

```diff
@@ -305,6 +305,29 @@ public void When_MappingBundleWithLifestyle_Expect_AdditionalInformation() {
         assertThat(actualHeaders).isEqualTo(expectedHeaders);
     }
 
+    @Test
+    public void When_MappingBundleFromMedicus_Expect_AdditionalInformation() {
+        var jsonFile = readResourceFile(String.format(BUNDLE_RESOURCE_DIRECTORY + "/%s.json",
+                "medicus"));
+        var bundle = fhirParser.parseResource(jsonFile, Bundle.class);
+
+        Map<String, String> expectedHeaders = new HashMap<>() {
+            {
+                put("Clinical Observations and Findings", "ObservationsHeader");
+                put("Diagnoses", "DiagnosesHeader");
+                put("Personal Preferences", "PreferencesHeader");
+                put("Problems and Issues", "ProblemsHeader");
+            }
+        };
+
+        var result = GpSummary.isBundleWithAdditionalInformation(bundle);
```

Comment:
I know this isn't part of this PR, but having a method `isBlah` return anything other than a boolean is confusing to me.